<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 57, 57);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>3</y>
      <width>1915</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>1860</x>
       <y>7</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>192.168.1.3</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="customPlot_1" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>365</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>202</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_3" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>40</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_4" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>527</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_5" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>689</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>180</y>
      <width>315</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;
color: rgb(60, 145, 202);
 border-radius: 30px;
    border-style: solid;
 border: 2px solid #555;
border-color: rgb(119, 255, 82);
  </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="down">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="left">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>86</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
 
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>87</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="horen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>87</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UP">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>16</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
  	
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>1602</x>
      <y>568</y>
      <width>315</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="throtal">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>10</y>
       <width>310</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;
color: rgb(60, 145, 202);
 border-radius: 30px;
    border-style: solid;
 border: 2px solid #555;
border-color: rgb(119, 255, 82);
  </string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>480</y>
      <width>155</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;
color: rgb(60, 145, 202);
 border-radius: 30px;
    border-style: solid;
 border: 2px solid #555;
border-color: rgb(119, 255, 82);
  </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="light">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="light_2">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>8</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
 
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>480</y>
      <width>155</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;
color: rgb(60, 145, 202);
 border-radius: 30px;
    border-style: solid;
 border: 2px solid #555;
border-color: rgb(119, 255, 82);
  </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="sos_2">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
 
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sos">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

	
    color: #333;
    border: 5px solid #555;
    border-radius: 30px;
    border-style: solid;
	border-color: rgb(119, 255, 82);

  
	background-color:  rgb(255, 255, 255);
    padding: 5px;
    }

QPushButton:hover {
    background: rgb(218, 209, 209);
    }

QPushButton:pressed {
    border-style: inset;
    background:  rgb(255, 255, 255);
    }</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>440</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;

 border-radius: 10px;
    border-style: solid;
 border: 2px solid #555;
border-color: rgb(119, 255, 82);
color: rgb(119, 255, 82);
  </string>
    </property>
    <property name="text">
     <string>     LIGHT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>440</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 0);

font: 12pt &quot;Segoe UI&quot;;
color: rgb(60, 145, 202);
 border-radius: 10px;
    border-style: solid;
 border: 2px solid #555;
color: rgb(119, 255, 82);
border-color: rgb(119, 255, 82);
  </string>
    </property>
    <property name="text">
     <string>      SOS</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>557</y>
      <width>1261</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_6" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>853</y>
      <width>322</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>41</y>
      <width>632</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>964</x>
      <y>42</y>
      <width>632</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>right</tabstop>
  <tabstop>down</tabstop>
  <tabstop>UP</tabstop>
  <tabstop>horen</tabstop>
  <tabstop>throtal</tabstop>
  <tabstop>sos</tabstop>
  <tabstop>light</tabstop>
  <tabstop>left</tabstop>
  <tabstop>light_2</tabstop>
  <tabstop>sos_2</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
